compute/client_library/ingredients/instances/ip_address/get_vm_address.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# flake8: noqa
+
+
+from google.cloud import compute_v1
+from enum import Enum
+
+
+# <INGREDIENT get_instance_ip_address>
+class IPType(Enum):
+    INTERNAL = "internal"
+    EXTERNAL = "external"
+    IP_V6 = "ipv6"
+
+
+def get_instance_ip_address(instance: compute_v1.Instance, ip_type: IPType) -> str:
```

Comment:
What if the instance has multiple interfaces and addresses? We should return a list.

Reply:
good point, fixed